### ISSUE TITLE
RFC: Logging

### ISSUE THREAD

**Issue by ep12** [qtile/qtile#1672]
@ramnes suggested that the log might be too verbose sometimes (See #1657).
Let's open this for a discussion about logging and collect some ideas.
+ [ ] make `libqtile sensors.py:_format_sensors_output():L109` a `debug` log entry
+ [ ] make `libqtile clock.py:__init__():L61` a `debug` log entry
+ [ ] make `libqtile.core.Qtile.cmd_restart` `info` us why it is restarting:
```python
import inspect
...
    def cmd_restart(self):
        f = inspect.currentframe().f_back
        if f is not None:
            logger.info("Restart requested: %s:%s", f.f_code.co_filename, f.f_lineno)
```

**Comment by zordsdavini** [comment]
can you update description with what we have now

**Comment by ep12** [comment]
What do we have now?

**Comment by pawamoy** [comment]
That's the question :smile: I think @zordsdavini would like to know with what level these things are currently logged.

**Comment by zordsdavini** [comment]
checked the code:
-  `libqtile sensors.py:_format_sensors_output():L109` now we have `info`
- `libqtile clock.py:__init__():L61` now `info`
- `libqtile.core.Qtile.cmd_restart` now `none`

**Comment by ep12** [comment]
Jep, that's the problem. The `_format_sensors_output` should really be `debug`, not `info`! Same for the clock. Info is more important than repeated temperature data, so that should change, I think.

**Comment by ramnes** [comment]
```
2020-05-05 17:54:10,332 WARNING libqtile xcbq.py:set_property():L687 X error in SetProperty (wid=33715743, prop='WM_STATE'), ignoring
```

this should be debug

**Comment by ramnes** [comment]
```
Traceback (most recent call last):
  File "/home/ramnes/perso/qtile/libqtile/backend/x11/xcore.py", line 245, in _xpoll
    ret = target(event)
  File "/home/ramnes/perso/qtile/libqtile/backend/x11/xcore.py", line 493, in handle_KeyPress
    self.qtile.process_key_event(keysym, event.state & self._valid_mask)
  File "/home/ramnes/perso/qtile/libqtile/core/manager.py", line 349, in process_key_event
    key = self.keys_map[(keysym, mask)]
KeyError: (65515, 0)
```
this should not raise an error but trigger a debug or warning log, I guess?

**Comment by tych0** [comment]
That second one already has a log, the code is just not using it. I noticed this the other day, let me send a patch.

**Comment by ramnes** [comment]
```
2020-05-12 16:31:40,456 ERROR libqtile.widget.pulse_volume pulse_volume.py:on_connection_change():L104 Connection to pulseaudio has been terminated
```
this happens when restarting qtile with the PulseVolume widget in use, it should not (or not an error, at least)

**Comment by ramnes** [comment]
Fixed in d297e46. I didn't add the extra info in `cmd_restart` because @DorianGray is working on restarts at the moment, but feel free to open a PR for this once #2046 is merged. :)
